commands/beer-lookup.js

```diff
@@ -23,66 +37,70 @@ exports.brew = {
 					let thumbnail = '';
 					if (typeof result.labels !== 'undefined') {
 						thumbnail = result.labels.medium;
-					};
+					}
+
 					if (typeof result.images !== 'undefined') {
 						thumbnail = result.images.squareMedium;
 					}
+
 					if (typeof result.style !== 'undefined') {
-							fields.push({
-									name: `Style: ${result.style.name}`,
-									value: result.style.description
-							});
-					};
+						fields.push({
+							name: `Style: ${result.style.name}`,
+							value: result.style.description
+						});
+					}
 					if (typeof result.abv !== 'undefined') {
-							fields.push({
-									name: 'ABV (Alcohol By Volume)',
-									value: `${result.abv}%`,
-									inline: true
-							});
-					};
+						fields.push({
+							name: 'ABV (Alcohol By Volume)',
+							value: `${result.abv}%`,
+							inline: true
+						});
+					}
 					if (typeof result.ibu !== 'undefined') {
-							fields.push({
-									name: 'IBU (International Bitterness Units)',
-									value: `${result.ibu}/100`,
-									inline: true
-							});
-					};
+						fields.push({
+							name: 'IBU (International Bitterness Units)',
+							value: `${result.ibu}/100`,
+							inline: true
+						});
+					}
+
 					if (typeof result.website !== 'undefined') {
-							fields.push({
-									name: 'Website',
-									value: result.website,
-									inline: true
-							});
-					};
+						fields.push({
+							name: 'Website',
+							value: result.website,
+							inline: true
+						});
+					}
+
 					if (typeof result.established !== 'undefined') {
-							fields.push({
-									name: 'Year Established',
-									value: result.established,
-									inline: true
-							});
+						fields.push({
+							name: 'Year Established',
+							value: result.established,
+							inline: true
+						});
 					};
 
 					msg.channel.send({
-							embed: {
-									color: GReYBot.Config.defaultEmbedColor,
-									title: result.name,
-									author: {
-										name: 'BreweryDB',
-										url: 'http://www.brewerydb.com/',
-										icon_url: 'https://emojipedia-us.s3.amazonaws.com/thumbs/120/twitter/103/beer-mug_1f37a.png'
-									},
-									thumbnail: {
-											url: thumbnail
-									},
-									description: `\n${result.description}\n\n`,
-									fields: fields
-							}
+						embed: {
+							color: GReYBot.Config.defaultEmbedColor,
+							title: result.name,
+							author: {
+								name: 'BreweryDB',
+								url: 'http://www.brewerydb.com/',
+								icon_url: 'https://emojipedia-us.s3.amazonaws.com/thumbs/120/twitter/103/beer-mug_1f37a.png'
+							},
+							thumbnail: {
+								url: thumbnail
+							},
+							description: `\n${result.description}\n\n`,
+							fields: fields
+						}
 					});
 				} else {
 					msg.channel.send({
 						embed: {
```

Comment:
if we're going to style the embed for a mistake in typing, we should style the response for no description